### DISCUSSION TITLE
Mobile app accessibility

### DISCUSSION THREAD

**Discussion by shawna-slh** [w3c/wai-mission#7] [Specific, miscellaneous]
In some parts of the world, mobile apps, super apps, and miniapps are much more prevalent than "traditional" web and browsers.

W3C work is introduced and linked in [Mobile Accessibility at W3C](https://www.w3.org/WAI/standards-guidelines/mobile/).

Where should we be focusing work on mobile app accessibility?
* Clearer coverage in [WCAG2ICT](https://www.w3.org/WAI/standards-guidelines/wcag/non-web-ict/)?
* Updating [Mobile Accessibility: How WCAG 2.0 and Other W3C/WAI Guidelines Apply to Mobile](https://www.w3.org/TR/mobile-accessibility-mapping/) for WCAG 2.2?
* Developing Mobile App Accessibility [supplemental guidance](https://www.w3.org/WAI/WCAG2/supplemental/about/)?
* [WCAG 3](https://www.w3.org/WAI/standards-guidelines/wcag/wcag3-intro/) though it's years away?
* Other?

**Are these sufficient to meet the world's needs for app accessibility guidance and standards?**

How best to liaise with other potential standards development around app accessibility?

**Comment by lbeaze** [comment]
I'm wondering if we can add non-web techniques where appropriate. 

For example, [H48](https://www.w3.org/WAI/WCAG22/Techniques/html/H48.html) (for [1.3.1](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html)) described web lists. Could we add lists for other platforms? 

I've got a library of responses for auditors who apply their expectations of web lists to the Android (for example). Not all platform controls work with screen readers they way we're used to them working with a web list. 

If we expanded our techniques we could more quickly help tool makers and auditors understand how to think about the Criterion.  

This could also keep people in the Understanding Docs - instead of then reference another source (ex: WCACG2ICT) for more info.
